docs/configuration.asciidoc

```diff
@@ -94,7 +94,7 @@ In this case, you can set the log level of the agent with `ElasticApm:LogLevel`,
 This is used to keep all the errors and transactions of your service together
 and is the primary filter in the Elastic APM user interface.
 
-NOTE: The service name must conform to this regular expression: ^[a-zA-Z0-9 _-]+$. In less regexy terms: Your service name must only contain characters from the ASCII alphabet, numbers, dashes, underscores and spaces.
+NOTE: The service name must conform to this regular expression: ^[a-zA-Z0-9 _-]+$. In less regexy terms: Your service name must only contain characters from the ASCII alphabet, numbers, dashes, underscores and spaces. Characters in service name which don't fit regular expression will be replaced to `__` symbol.
```

Comment:
I think you have underscores in
> will be replaced to `__` symbol

Reply:
Yeap, because one underscore isn't rendered with `asciidoc`